src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -141,6 +146,9 @@ m_cursorGrabbed   (false)
 
     if (m_handle)
     {
+        if (handleCount++ == 0)
```

Comment:
I strongly think branching conditions should be side effect free; so yeah, I've an issue with this. 😛